pandas-stubs/core/reshape/encoding.pyi

```diff
@@ -0,0 +1,31 @@
+from typing import (
+    Hashable,
+    Iterable,
+)
+
+from pandas import (
+    DataFrame,
+    Series,
+)
+
+from pandas._typing import (
+    ArrayLike,
+    Dtype,
+    HashableT,
+)
+
+def get_dummies(
+    data: ArrayLike | DataFrame | Series,
+    prefix: str | Iterable[str] | dict[str, str] | None = ...,
```

Comment:
For the `dict` case, docs say "prefix can be a dictionary mapping column names to prefixes", so maybe this should be `dict[Hashable, str]`

Reply:
Fixed.

Reply:
thanks